CHANGELOG.rst

```diff
@@ -5,6 +5,10 @@ These are notable changes in edx-platform.  This is a rolling list of changes,
 in roughly chronological order, most recent first.  Add your entries at or near
 the top.  Include a label indicating the component affected.
 
+LMS: Assign cohorts to content groups in the instructor dashboard.  TNL-653
```

Comment:
Since @andy-armstrong pointed out how sporadically this file is updated, I'd be happy to have a single entry to make things clearer.